src/Microsoft.Health.Fhir.SqlServer/Registration/FhirServerBuilderSqlServerRegistrationExtensions.cs

```diff
@@ -30,7 +33,22 @@ public static IFhirServerBuilder AddSqlServer(this IFhirServerBuilder fhirServer
             services.AddSqlServerBase<SchemaVersion>(configureAction);
             services.AddSqlServerApi();
 
-            services.Add(provider => new SchemaInformation((int)SchemaVersion.V1, (int)SchemaVersion.V2))
+            services.Add(provider => new SchemaInformation((int)SchemaVersion.V1, (int)SchemaVersion.V3))
+                .Singleton()
+                .AsSelf()
+                .AsImplementedInterfaces();
+
+            services.Add<SqlServerStatusRegistry>()
+                .Singleton()
+                .AsSelf()
+                .AsImplementedInterfaces();
+
+            services.Add<SqlConnectionWrapperFactory>()
+                .Singleton()
+                .AsSelf()
+                .AsImplementedInterfaces();
```

Comment:
Was this missing?

Reply:
It's used in a few other services (`SqlServerSearchService`, `SqlServerFhirOperationDataStore`, `SqlServerFhirDataStore`), but they are all added as scoped services and not a singleton

Reply:
I think this entry should be removed, we shouldn't re-register this service using a different scope

Reply:
Right, makes sense. I took this out in 725eefb.